frontend/components/dashboard/DashboardHeader.tsx

```diff
@@ -42,11 +42,16 @@ export const DashboardHeader = ({
               : CREATE_POLL_ROUTE
           }
         >
-          <a>
-            <Button color={colors.GREEN} round style={{ margin: '0 0 0 1rem' }}>
+          <Button
+            className="flex items-center justify-center"
+            color={colors.GREEN}
```

Comment:
where is colors coming from? Should add the colors we are using to the tailwind config so we can just use them directly in the classname?

Reply:
Yeah I am planning on this eventually, right now it's more of a gradual adoption of Tailwind, so I want to get everything in place first.